app/core/Tracker2/DriftTracker.js

```diff
@@ -111,11 +112,14 @@ export default class DriftTracker extends BaseTracker {
   }
 
   get isChatEnabled () {
-    return !this.onPlayPage && !this.store.getters['me/isStudent']
+    return !this.onPlayPage && !this.store.getters['me/isStudent'] && !this.store.getters['me/isHomePlayer']  // && !this.disableAllTracking
```

Comment:
Would we want to keep the `disableAllTracking` for the cookieConsent check, smoketest user checks?

Reply:
Problem: Drift is used as support tool, and `disableAllTracking` respects browser `doNotTrack`, but we do want the support variant of it to show up when `doNotTrack` is set. So we have mixed behavior with Drift in that case: don't track page views and events, but do allow support chat with limited context sent to Drift.

Well, perhaps we could respect it anyway and just rely on the new Contact option in the footer for email support?

Reply:
Fair point. I'm fine either way.